test/service/test_request_validation.py

```diff
@@ -282,6 +282,7 @@ def test_bdbag_manifest(self):
             self.assertEqual(200, response.status_code, 'Unable to download manifest')
             with ZipFile(BytesIO(response.content), 'r') as zip_fh:
                 zip_fh.extractall(zip_dir)
+                self.assertTrue(['manifest' == first(name.split(os.path.sep)) for name in zip_fh.namelist()])
```

Comment:
I don't think the separator in a ZIP entry is OS-dependent. It's always going to be `/`.

Reply:
And your assertTrue() doesn't work as intended:

```
from unittest import TestCase
TestCase().assertTrue([False])
```
No exception is raised.